### PR TITLE
feature(Ecosystem): Add Ecosystem

### PR DESCRIPTION
**Pull request by TheGreatAxios**
- Initialize Ecosystem Section
   - Initialize Delegators Section
   - Initialize Governance Section
   - Initialize Validators Section

### CODE REVIEW

File: src/content/docs/ecosystem/index.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    order: 2
+---
+
+import { LinkCard, CardGrid } from '@astrojs/starlight/components';
+
+## The SKALE Ecosystem
+<CardGrid>
+    <LinkCard
+        title="Builders"
+        href="/builders"
```
Comment (manuelbarbas) [inline src/content/docs/ecosystem/index.mdx:14]: Was this builder card placed here on purpose? I would remove it but let me know your thoughts

File: src/content/docs/ecosystem/governance/overview.mdx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Web3 technologies, like SKALE’s protocols and chains, are initially developed by companies but progressively become decentralized with
+the growth of a user and stakeholder community. This progressive decentralization typically involves DAO formation and launching a
+process by which certain stake holders may vote. In this way, the SKALE DAO, a decentralized autonomous organization, has
+decision-making authority over the SKALE Network and its associated technologies. To steer the SKALE DAO, staking SKL tokens signifies
+DAO membership. Staked token holders can vote on DAO proposals.
+
+## How can I participate?
+
+- [Delegate to a SKALE validator](/ecosystem/delegations/overview).
```
Comment (manuelbarbas) [inline src/content/docs/ecosystem/governance/overview.mdx:40]: This link is incorrect. It should redirect to one of the two:
- ecosystem/delegators/overview: this is the correct path of the delegators overview page
- the portal for perform delegations: https://portal.skale.space/staking

File: src/content/docs/ecosystem/delegators/overview.mdx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+tokens for/into the SKALE Network. Transfers and approvals work the same
+in ERC-777 as in ERC-20 -- the only difference is that ERC-777
+allows for certain functions and checks before transfers are executed.
+The SKL token uses this pre-transfer function to conduct checks
+to ensure that tokens are not actively delegated.
+
+### Important Information
+
+- The SKALE Token Address on the Ethereum Mainnet is: [0x00c83aeCC790e8a4453e5dD3B0B4b3680501a7A7](https://etherscan.io/address/0x00c83aeCC790e8a4453e5dD3B0B4b3680501a7A7)
+- GitHub Contract Code can be found [here]()
```
Comment (manuelbarbas) [inline src/content/docs/ecosystem/delegators/overview.mdx:31]: SKL contract code: https://github.com/skalenetwork/skale-manager/blob/b039ebebb16a1a2a97a707d7ba6a0f2636413f62/contracts/SkaleToken.sol

File: src/content/docs/ecosystem/index.mdx (hunk 1 above)
Comment (TheGreatAxios) [reply src/content/docs/ecosystem/index.mdx:14]: Yes -- placed on purpose as builders are part of the ecosystem.

File: src/content/docs/ecosystem/governance/overview.mdx (hunk 2 above)
Comment (TheGreatAxios) [reply src/content/docs/ecosystem/governance/overview.mdx:40]: Agree -- will fix

File: src/content/docs/ecosystem/delegators/overview.mdx (hunk 3 above)
Comment (TheGreatAxios) [reply src/content/docs/ecosystem/delegators/overview.mdx:31]: Thank you

File: src/content/docs/ecosystem/governance/overview.mdx (hunk 2 above)
Comment (TheGreatAxios) [reply src/content/docs/ecosystem/governance/overview.mdx]: Resolved by [527cf3e](https://github.com/skalenetwork/docs.skale.space/pull/127/commits/527cf3ef3f69cd2ecfbbc70e85576610d7ee2377)

File: src/content/docs/ecosystem/delegators/overview.mdx (hunk 3 above)
Comment (TheGreatAxios) [reply src/content/docs/ecosystem/delegators/overview.mdx]: Resolved by [578991a](https://github.com/skalenetwork/docs.skale.space/pull/127/commits/578991ae0b2a57b76bcaede88d85ad33dff249bf)

**Review by manuelbarbas — APPROVED** [review]

**Pull request merged**